test/runtests.jl

```diff
@@ -1,26 +1,28 @@
 using Base: operator_associativity
 
-include("recursive.jl")
```

Comment:
I would revert this change as the original ordering  in runtests is useful for debugging because it better captures the rough dependency order of functionality for the tests

Reply:
Oh, I thought the alphabetical order was just better organized for people to look for existing tests and insert new ones. Also, the other day I found a test that was included twice (`diffusion2d.jl`) and I thought that if they were alphabetized it would be clearer to avoid double entries.

Reply:
it might be good to add a comment about why the ordering is the way it is for the future

Reply:
Done.